OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
Please lets not go back to the bad-old days of layering platform-specific workarounds in common code :(
You could do the linux-specific manipulations in a launch-replay shellscript wrapper.  Compare with [OpenRA.m#L80](https://github.com/pchote/OpenRA/blob/057528ebc991a302c24fe1be134a9d46c8893b79/packaging/osx/OpenRA.m#L80) for OSX.